Networking/Sources/Networking/Base/Endpoint.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  File.swift
```

Comment:
That title is left by default, it will be cool if you update is so it corresponds to the actual file name